tracer/test/benchmarks/Benchmarks.Trace/Log4netBenchmark.cs

```diff
@@ -51,15 +64,50 @@ static Log4netBenchmark()
             Logger = log4net.LogManager.GetLogger(typeof(Log4netBenchmark));
         }
 
+        [GlobalSetup]
+        public void GlobalSetup()
+        {
+            InitializeTracer();
+            InitializeLogger(useDatadogAppender: true);
+        }
+
         [Benchmark]
-        public void EnrichedLog()
+        public void WriteLogs()
```

Comment:
Renaming this will break the existing dashboards, as well as our benchmark comparison tool (only for this branch of course). Maybe it's not worth renaming it?
```suggestion
        public void EnrichedLog()
```

Reply:
Reverted the name in aff538a61d2115e82b5c3112ba5760cdb491234d